lib/create-github-pr.js

```diff
@@ -89,24 +89,6 @@ module.exports = function createPullRequest(out) {
 
       return chain;
     })
-    .then(() => addOrUpdateFile(`fixtures/register.json`, `register.json`, oldFileContent => {
```

Comment:
All previous `addOrUpdateFile` calls add `[skip ci]` to their commit messages. Since this last one which doesn't is now removed, CI won't be triggered at all. So the last fixture in the loop above needs to pass `false` to the function instead.